### DISCUSSION TITLE
As an organization we want Copilot Subscriptions for our developers

### DISCUSSION THREAD

**Discussion by bitflorist** [community/community#33183] [Copilot Conversations] [answered]
Hi,
we have 15 developers+ and would like to enable copilot for our developers.
We have not found a solution yet, on how we can enable copilot for our devs.
Does anyone have a clue, how to do that?
The only option we found so far is enabling the personal account.
Our compliance does not allow Personal Accounts to be charged with the company credit card.

best 

- radosch

**Comment by iamcjwilliams** [comment, marked as answer]
Hello @radosch currently, Copilot is only available for individual user accounts and organizations aren't able to purchase/manage Copilot for their members just yet. However, you can sign up on the waitlist here: https://github.com/features/copilot/org_signup
